src/ui/markdown.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var Util = require('../util');
+
+var g = Util.getGlobal(),
+    _t = Util.gettext;
+
+function createMarkdownPlugin () {
+    var converter = null;
+
+    if (typeof g.Showdown !== "undefined" &&
+        g.Showdown !== null &&
+        typeof g.Showdown.converter !== "undefined" &&
+        g.Showdown.converter !== null) {
+        converter = new g.Showdown.converter();
```

Comment:
Looking at the history, I added that check, but didn't explain why. Sorry. Looking at the Showdown docs this function seems to be part of the API so perhaps a better check would be.

```
if (g.Showdown && typeof g.Showdown.converter === 'function') {
```